refarch/aws-native/cdk.json

```diff
@@ -2,11 +2,18 @@
   "app": "python3 app.py",
   "context": {
     "@aws-cdk/core:enableStackNameDuplicates": "true",
+    "@aws-cdk/core:newStyleStackSynthesis":  "true",
     "aws-cdk:enableDiffNoFail": "true",
     "EnableBatch": "true",
     "EnableDWH": "true",
     "EnableDataviz": "true",
     "EnableStreaming": "true",
+    "EnableCICD": "false",
+    "RepositoryName": "<REPOSITORY_NAME>",
```

Comment:
should we move those to context ? or sub object for clarification ?

Reply:
Not sure if I understand you here.  It's already in `"context"`, isn't it?

Do you mean `cdk.context.json`? I understand that context is for runtime information (e.g. account id or AZ) [link](https://docs.aws.amazon.com/cdk/api/latest/docs/pipelines-readme.html#context-lookups). 

When you say sub-object. Do you mean something like this?

```
"EnableCICD: "true",
"CICDParameters": {
  "RepositoryName": "<REPOSITORY_NAME>","RepositoryBranch": "<REPOSITORY_BRANCH>",
  "ConnectionArn": "<CONNECTION_ARN>",
}
```

Reply:
As discussed over the call, I grouped this into the same object for readability.